packages/template-retail-react-app/app/utils/product-utils.js

```diff
@@ -33,3 +33,21 @@ export const getDisplayVariationValues = (variationAttributes, values = {}) => {
     }, {})
     return returnVal
 }
+
+/**
+ * This function extract the promotional price from a product. If there are more than one price, the smallest price will be picked
+ * @param {object} product - product detail object
+ * @returns {{discountPrice: (number|number), basePrice: *}}
```

Comment:
```suggestion
 * @returns {{discountPrice: number, basePrice: number}}
```
(I assume `basePrice` is a number, but I guess it could be a string?)